_posts/2015-10-30-blend.md

```diff
@@ -0,0 +1,220 @@
+---
+layout: post
+title: "Blend Web Mix 2015 : On y étais !"
```

Comment:
:sweat_smile: I don't know what you're talking about :p I'm gonna change this now, it just can't wait x)